pkg/cluster/internal/providers/docker/network_test.go

```diff
@@ -154,9 +154,7 @@ func Test_sortNetworkInspectEntries(t *testing.T) {
 		tc := tc
 		t.Run(tc.Name, func(t *testing.T) {
 			toSort := make([]networkInspectEntry, len(tc.Networks))
-			for i := range tc.Networks {
-				toSort[i] = tc.Networks[i]
-			}
+			copy(toSort, tc.Networks)
```

Comment:
TODO: As discussed in commit message, we should consider disabling this linter instead, this is an inaccurate error and you might introduce a bug by switching.
We're fine in this case though.